helm/logging-operator/templates/configmap.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "resource.default.name"  . }}
+  namespace: {{ include "resource.default.namespace"  . }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+data:
+  config.yml: |
+    server:
+      enable:
+        debug:
+          server: true
+      listen:
+        address: 'http://0.0.0.0:8000'
+    service:
+      alertmanager:
+        address: {{ .Values.alertmanagerAddress }}
```

Comment:
I think alertmanager is not needed here :D

Reply:
True !